Sources/Picture/Internal/SingleImageView.swift

```diff
@@ -0,0 +1,16 @@
+import SwiftUI
+
+// TODO: [P-8](https://github.com/0xOpenBytes/Picture/issues/8)
+public struct SingleImageView: View {
+    let image: Image
+
+    public var body: some View {
+        image
```

Comment:
Does this need `.resizable` on it? And perhaps ` .aspectRatio(contentMode: .fit)` or a way to set it?

Reply:
Ah I was calling it wrong, never mind